pkg/xds/topology/dataplanes.go

```diff
@@ -29,8 +28,11 @@ func GetDataplanes(ctx context.Context, rm manager.ReadOnlyResourceManager, mesh
 	return rv, nil
 }
 
-func ResolveAddress(dataplane *core_mesh.DataplaneResource) error {
-	ips, err := dns.LookupIP(dataplane.Spec.Networking.Address)
+// ResolveAddress resolves 'dataplane.networking.address' if it has DNS name in it. This is a crucial feature for
+// some environments specifically AWS ECS. Dataplane resource has to be created before running Kuma DP, but IP address
+// will be assigned only after container's start. Being able to set dp's address as a DNS name solves this problem
```

Comment:
yeah, but then again - why are we resolving? :)

Add info that "Hostname cannot be used in EDS"